src/main/webapp/js/grapheditor/Shapes.js

```diff
@@ -3621,11 +3621,68 @@ function getShape(cell) {
 		c.fillAndStroke();
 	});
 
-  mxGraphModel.prototype.valueForCellChanged = function(cell, value) {
-    cell.value = value;
-    this.setStyle(cell, `${cell.getStyle()}name=${value};`)
+  function tableValue(style) {
+    permission = " (Contol: ";
+    if(style.cpiCreate === 1) {
+      permission += "C";
+    }
+    if(style.cpiRead === 1) {
+      permission += "R";
+    }
+    if(style.cpiUpdate === 1) {
+      permission += "U";
+    }
+    if(style.cpiDelete === 1) {
+      permission += "D";
+    }
+    permission += " Dpath: ";
+    if(style.dpCreate === 1) {
+      permission += "C";
+    }
+    if(style.dpRead === 1) {
+      permission += "R";
+    }
+    if(style.dpUpdate === 1) {
+      permission += "U";
+    }
+    if(style.dpDelete === 1) {
+      permission += "D";
+    }
+    permission += ")";
```

Comment:
1. Typo (Contol -> Control)
2. I would make it a simple function:

```suggestion
    const makePermissionString(create, read, update, delete) {
      return [
          create === 1 ? 'C' : '',
          read === 1 ? 'R' : '',
          update === 1 ? 'U' : '',
          delete === 1 ? 'D' : '',
      ].join('');
    }
    permission = " (Control: " + makePermissionString(
      style.cpiCreate,
      style.cpiRead,
      style.cpiUpdate,
      style.cpiDelete,
    );
    permission += " Dpath: " + makePermissionString(
      style.dpCreate,
      style.dpRead,
      style.dpUpdate,
      style.dpDelete,
    );
    permission += ")";
```